app/jobs/socure_image_retrieval_job.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+class SocureImageRetrievalJob < ApplicationJob
+  queue_as :high_socure_docv
```

Comment:
is this right?

Reply:
I don't think we'd want these jobs in the same queue as high-priority idv jobs